lib/src/widgets/custom_scroll_behavior.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class CustomScrollBehavior extends ScrollBehavior {
+  const CustomScrollBehavior({
+    this.bottomPadding = 0,
+  });
+
+  /// The amount of space the scrollbar should have before reaching the bottom.
+  final double bottomPadding;
+
+  @override
+  Widget buildScrollbar(
+      BuildContext context, Widget child, ScrollableDetails details) {
+    switch (getPlatform(context)) {
```

Comment:
Can you please convert this to switch expression?

Reply:
This is the way it is implemented in the parent class. I don't think the switch expression would help here, considering I would have to be very verbose.